team-agreements/working-agreements/readme.md

```diff
@@ -1,15 +1,63 @@
+- We are not expected to answer emails past 6PM, on weekends on when we are on holidays or vacation.
+- We work in different time zones and respsect this, especially when setting up recurring meetings.
+- We record meetings when possible, so that team members who could not attend live can listen later.
+
+## Quality and not quantity
+
+- We, together on a [Definition of Done](../definition-of-done/readme.md) for our user story's and sprints and live by it.
+- We follow engineering best practices like the [Code With Engineering Playbook](https://github.com/microsoft/code-with-engineering-playbook)
+
 ## Scrum rhythm
 
-## Backlog management
+| Activity | When | Duration | Who | Accountable | Goal |
+|-|-|-|-|-|-|
+| [Project Standup](./sprint-planning/../../../stand-ups/readme.md) | Tue-Fri 9AM | 15 min | Dev Team, PM | Scrum Master | What has been accomplished, next steps, blockers |
+| Sprint Demo | Monday 9AM | 1 hour | Dev Team, PM, PO | PM and Dev Lead | Present work done and sign off on user story completion |
+| [Sprint Retro](./../../retrospectives/readme.md) | Monday 10PM | 1 hour | Dev Team, PM | Scrum Master | Dev Teams shares learnings and what can be improved |
+| [Sprint Planning](./../../sprint-planning/readme.md) | Monday 11PM | 1 hour | Dev Team, PM, Customer PO | PM | Size and plan user stories for the sprint |
+| Task Creation | After Sprint Planning | - | Dev Team, PM | Dev Lead | Create tasks to clarify and determine velocity |
+| [Backlog grooming](../../backlog-management/grooming/readme.md) | Wednesday 2PM | 1 hour | PM, PO, Dev Lead, Scrum Master, Dev Team | PM | Prepare for next sprint and ensure that stories are ready for next sprint. |
 
-## Code management
+## Backlog management
 
-## Design reviews
+- We work together on a [Definition of Ready](../definition-of-ready/readme.md) and all user stories assigned to a sprint need to follow this
+- We communicate what we are working on through the board
+- We assign ourselves a task when we are ready to work on it (not before) and move it to active
+- We capture any work we do related to the project in a user story/task
+- We close our tasks/user stories only when they are done (as described in the [Definition of Done](../definition-of-done/readme.md))
+- We work with the PM if we want to add a new user story to the sprint
+- If we add new tasks to the board, we make sure it matches the acceptance criteria of the user story (to avoid scope creep). If it doesn't match the acceptance criteria we should discuss with the PM to see if we need a new user story for the task or if we should adjust the acceptance criteria.
 
-In this section, we document:
+## Code management
 
-- When we require a design review
-- How we decide who participates (including external stakeholders)
-- Where we capture the decisions made during the review
+- We follow the git flow branch naming convention for branches and identify the task number eg. `feature/123-add-working-agreement`
```

Comment:
Most engagements we were working on had established git branch naming conventions and we are asked to follow those. Might also be a suggestion, if there are no established rules?

Reply:
added a note at the top saying that this is also to be adjusted to fit team needs